CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added the ability to run beams from the CLI for Generic Unix platform (it was already possible with nodejs and emscripten).
 - Added support for 'erlang:--/2'.
 - Added preliminary support for ESP32P4 (no networking support yet).
+- Added support for list insertion in 'ets:insert/2'.
```

Comment:
I think it got by mistake in the wrong section of the changelog file.
Changed aimed to `main` branch should be under `## Unreleased` (not in `## [0.6.6] - Unreleased` section).
The right place should be just below `Added `supervisor:terminate_child/2`, `supervisor:restart_child/2`.